### ISSUE TITLE
UI renders literal "i18n"

### ISSUE THREAD

**Issue by futape** [neos/neos-ui#2005]
### Description

In the inspector, a literal "i18n" label is rendered for a select box editor option. Apparently UI doesn't understand that it should try to find a translation since event if a matching translation wouldn't be found, the translation shorthand should be rendered instead.

### Steps to Reproduce

1. I defined a mixin defining one property:

   ```yaml
    brandColor:
      type: string
      defaultValue: 'primary'
      ui:
        label: i18n
        reloadIfChanged: true
        inspector:
          group: style
          position: 10
          editor: 'Neos.Neos/Inspector/Editors/SelectBoxEditor'
          editorOptions:
            values:
              'primary':
                label: i18n
              'turquoise':
                label: i18n
              'gray-dark':
                label: i18n
              'gray-light':
                label: i18n
              'toyota-red':
                label: i18n
              'yellow':
                label: i18n
   ```
   here, everything works as expected. for example the "primary" label searches for the corresponding translation.
2. I defined a nodetype extending that mixin, adding an option to the property and changing the property's label:

   ```yaml
    brandColor:
      defaultValue: 'none'
      ui:
        label: 'i18n'
        inspector:
          editorOptions:
            values:
              'none':
                label: 'i18n'
   ```

  here, the `ui.label` is translated as expected, but for the `none` option the label isn't tried to be translated at all and instead "i18n" is rendered.

#### Expected behavior

If a matching translation for the label would be given, the translation should be rendered, otherwise the translation shorthand should be rendered, but never "i18n" should be rendered.

#### Actual behavior

![image](https://user-images.githubusercontent.com/4106712/42679302-c040d532-8681-11e8-9eac-da7c67e89758.png)

### Affected Versions

Neos: 4.0.6

UI: 1.1.2

**Comment by Lemmork** [comment]
Perhaps an addition to this.
I'm using the creationDialog and editorOptions labels are not translated. They only output "i18n".

**Comment by markusguenther** [comment]
<img width="324" alt="Screenshot 2021-10-12 at 23 12 06" src="https://user-images.githubusercontent.com/1014126/137030318-8b38270f-7dc0-4f6d-b66f-36202424d74e.png">

This has been fixed in `Neos.Neos` by @grebaldi 💙
